org.kde.klettres.json

```diff
@@ -0,0 +1,34 @@
+{
+    "id": "org.kde.klettres",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15",
+    "sdk": "org.kde.Sdk",
+    "command": "klettres",
+    "rename-icon": "klettres",
+    "finish-args": [
+        "--share=network", 
+        "--share=ipc", 
+        "--socket=x11", 
+        "--socket=wayland",
+	"--device=dri",
```

Comment:
nitpick, the indenting is off here now.

Reply:
ooops sorry, fix on https://github.com/flathub/flathub/pull/1805/commits/67ce54faa5abc79f5fe278042d85d871e15107ed